auto-ipsec/racoon/README.md

```diff
@@ -4,33 +4,33 @@ Using the certificate distribution mechanism built into keylime, you can automat
 
 ## Overview
 
-These scripts allow keylime nodes to create a virtual private network between them that is protected by cryptographic keys bootstrapped by keylime.  Keylime will also automatically remove failed nodes from the network.  The scripts work by leveraging two features of keylime: automatic certificate generation/delivery, and the ability run scripts provided in the payload after successful bootstrapping.  The following high level actions will happen:
+These scripts allow keylime agents to create a virtual private network between them that is protected by cryptographic keys bootstrapped by keylime.  Keylime will also automatically remove failed agents from the network.  The scripts work by leveraging two features of keylime: automatic certificate generation/delivery, and the ability run scripts provided in the payload after successful bootstrapping.  The following high level actions will happen:
```

Comment:
many of the same comments from the libreswan read me apply here.  wont' repeat

Reply:
Ditto again here -- needs to specify that the cloud machines/nodes (not Keylime) is communicating/blocked over IPsec.  The Keylime agent software itself never communicates over IPsec.